backend/src/backend/routes/books.py

```diff
@@ -109,6 +111,14 @@ async def create_book(book_payload: BookAddSchema):
         )
         await book.languages.add(language)
         await title.languages.add(language)
+    try:
+        # use via utils.xxx to accomodate monkeypatch in tests
+        await utils.remove_obsolete_books(
+            title_ident=title.ident,
+            flavour=book_payload.metadata.get("Flavour", ""),
+        )
+    except Exception as exc:
+        logger.exception(exc)
```

Comment:
Please add an error log entry in addition to the exception trace so it's easier to find